test/transaction.js

```diff
@@ -56,13 +56,29 @@ describe('Transaction', function () {
         var actual = Transaction.fromHex(txHex)
 
         assert.strictEqual(actual.toHex(), txHex)
+        if (!f.whex) {
+          // fromHex tolerates either format, so only test this if witness is not set
+          assert.strictEqual(actual.toLegacyBuffer().toString('hex'), txHex)
+        }
+        if (f.id) {
+          // id is always whatever's in the file
```

Comment:
Actually, removing that one since the getId test will test the witness if available.